src/main/content/_assets/js/builds.js

```diff
@@ -710,12 +711,15 @@ function validate_starter_inputs(event) {
                 if(EEVersionValue !== "None") {
                     curr_selected_mp_version = valuetoSelect = dependencies[d][dependencies[d].length - 1];
                 }
-                else{
-                    curr_selected_mp_version = valuetoSelect = starter_info[d].default;
-                }
                 var prev_selected_mp_version = options
                 .filter(':selected')
                 .text();
+                if ((EEVersionValue == "None")&&(prev_selected_mp_version == "None")) {
```

Comment:
```suggestion
                if((EEVersionValue === "None") && (prev_selected_mp_version === "None")) {
```

I would suggest always prefer `===` over `==` unless type conversion is needed. In this case, I believe the code is always dealing with String type and no type conversion is expected.

Reply:
Done